docs/user.md

```diff
@@ -181,3 +181,40 @@ spec:
     operator: Exists
     effect: NoSchedule
 ```
+
+## How to clone an existing PostgreSQL cluster
+
+To spin up a new cluster as a clone of the existing one, you need to provide a
+clone section in the spec:
+
+```yaml
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+
+metadata:
+  name: acid-test-cluster
+spec:
+  clone:
+    uid: "efd12e58-5786-11e8-b5a7-06148230260c"
+    cluster: "acid-batman"
+    timestamp: "2017-12-19T12:40:33+01:00"
+```
+
+Here `cluster` is a name of a target cluster that is going to be cloned. If
+`timestamp` is not empty, then a new cluster will be cloned from an S3 bucket,
```

Comment:
the operator does not create S3 buckets. Perhaps "a new cluster will be cloned from S3 ... "